src/test/ui/asm/bad-arch.rs

```diff
@@ -1,19 +0,0 @@
-// compile-flags: --target wasm32-unknown-unknown
-// needs-llvm-components: webassembly
-
-#![feature(no_core, lang_items, rustc_attrs)]
-#![no_core]
-
-#[rustc_builtin_macro]
-macro_rules! asm {
-    () => {};
-}
-#[lang = "sized"]
-trait Sized {}
-
-fn main() {
-    unsafe {
-        asm!("");
-        //~^ ERROR asm! is unsupported on this target
```

Comment:
Rather than deleting this test, can you change it to a different unsupported architecture? Maybe SPARC or something.